.github/workflows/ci.yml

```diff
@@ -73,6 +73,16 @@ jobs:
       runner-env: ${{ needs.setup.outputs.runner-env }}
       write-caches: ${{ needs.setup.outputs.write-caches }}
 
+  build_all_arm64:
```

Comment:
Please add new jobs to the `summary` job at the bottom of this file. That will make these jobs contribute to the overall build status: https://github.com/openxla/iree/blob/3a70ddae54bf62700d593e335c722464be1b4235/.github/workflows/ci.yml#L1082-L1101

Reply:
will do